bbc/simorgh
# WSTEAM1-972 - Uploader error screen

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-972

Overall changes
======
- Adds `ErrorScreen` component
- Sets page title to `Error: {title}` if the user lands on this screen
- Adds Storybook story for error screen
- Renames existing `Form` to `FormScreen` for naming consistency
- Renames existing `SuccessMessage` to `SuccessScreen` for naming consistency
- Unrelated fix to return correct page type for Webvitals as this was throwing console errors

Testing
======
1. Visit https://wsteam1-972-uploader-error-screen--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=pages-ugc-page--error-screen&viewMode=story
2. Confirm it renders correctly and aligns to a11y AAC / screen-reader guidelines
3. Check the page title starts with `Error:` (translations to follow)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
